pmlc/dialect/op_lib/OpLibWrapperGen.cpp

```diff
@@ -24,28 +27,175 @@
 using llvm::raw_ostream;
 using llvm::Record;
 using llvm::RecordKeeper;
+using mlir::GenRegistration;
 using mlir::StringRef;
+using mlir::tblgen::EnumAttr;
 using mlir::tblgen::Operator;
 
-static const char* TBLGEN_DEBUG = std::getenv("TBLGEN_DEBUG");
+namespace pmlc {
+namespace dialect {
+namespace op {
+
+using namespace pmlc::dialect::op;  // NOLINT [build/namespaces]
+
+namespace tblgen {
+
+static inline std::map<StringRef, StringRef> getAttributes(const Operator& op) {
```

Comment:
We probably don't need to `inline` these (they aren't one liners and `inline` is optional anyways, the compiler doesn't really need it since it's just a hint).